UI/index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+        UpMessenger
+    </title>
+    <meta charset="UTF-8">
+    <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+<header>
+    <div id="headerDiv">
+        <span><img id="logo"
+                   src="images/logo.jpg"
+                   alt="logo"></span>
+        <h3 class="headerH" id="home"><a href="index.html">Home</a></h3>
+        <h3 class="headerH"><a href="create.html">Create post</a></h3>
+        <h3 class="headerH" id="logout"><a href="index.html">Log out</a></h3>
+    </div>
+</header>
+<main>
+    <div id="userInfo">
+        <h3 id="userInfoName">LiverFan</h3>
+        <img id="userInfoAvatar"
+             src="images/liverpool.jpg"
+             alt="user avatar">
+        <h3 class="userInfoH3">Posts: 3</h3>
+        <h3 class="userInfoH3">Likes: 3</h3>
+        <h3 class="userInfoH3" id="favouriteHashtags">Favourite hashtags:</h3>
+        <p>
+        <div class="tag">#GoldenCup</div>
+        <div class="tag">#WAT3-0LIV</div>
+        <div class="tag">#Unbeaten</div>
+        <div class="tag">#Arsenal</div>
+        <div class="tag">#EPL</div>
+        </p>
+
+    </div>
+    <div id="posts">
+        <div class="postDiv">
```

Comment:
It's more common to use kebab-case for css classes. Also, such naming as `postDiv` is bad practice, cause there is no need to duplicate tag name in class name.